### PR TITLE
ApplicationAutoscaling ScheduledAction Resource

### PR DESCRIPTION
**Pull request by JohnPreston**
Creating new resource to manage Scheduled Action with CFN

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by kddejong — APPROVED** [review]

File: resources/ApplicationAutoscaling_ScheduledAction/template.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  Function:
+    Timeout: 180  # docker start-up times can be long for SAM CLI
+    MemorySize: 256
+
+Resources:
+  TypeFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: awscommunity_applicationautoscaling_scheduledaction.handlers.resource
+      Runtime: python3.7
```
Comment (mmaeng) [inline resources/ApplicationAutoscaling_ScheduledAction/template.yml:15]: Might want to switch this to a newer runtime of Python.  3.7 is set to be deprecated soonish.

File: resources/ApplicationAutoscaling_ScheduledAction/template.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      CodeUri: build/
+
+  TestEntrypoint:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: awscommunity_applicationautoscaling_scheduledaction.handlers.test_entrypoint
+      Runtime: python3.7
```
Comment (mmaeng) [inline resources/ApplicationAutoscaling_ScheduledAction/template.yml:22]: Might want to switch this to a newer runtime of Python.  3.7 is set to be deprecated soonish.

File: resources/ApplicationAutoscaling_ScheduledAction/Makefile
Comment (mmaeng) [inline resources/ApplicationAutoscaling_ScheduledAction/Makefile:1]: Nothing against your Makefile, it's just assumes very specific installs are in place.

File: resources/ApplicationAutoscaling_ScheduledAction/template.yml (hunk 2 above)
Comment (JohnPreston) [reply resources/ApplicationAutoscaling_ScheduledAction/template.yml:22]: Oops, I forgot to port this. I aligned to 3.9 as that's the latest I have seen `cfn init` offer for python.

**Review by mmaeng — APPROVED** [review]
LGTM!

**Comment by JohnPreston** [comment]
Rebased from main

**Review by mmaeng — APPROVED** [review]

**Pull request merged**